viennacl/linalg/detail/bisect/bisect_large.hpp

```
@@ -122,7 +123,8 @@ processResultDataLargeMatrix(ResultDataLarge<NumericT> &result,
 
       else
       {
-       // printf("pos_mult[%u] = %u\n", i, pos_mult[i]);
+
+        printf("error: linalg/detail/bisect_large.hpp: pos_mult[%u] = %u\n", i, pos_mult[i]);
```

This should throw an exception, not print messages to stdout